### PR TITLE
Alfoa/formatscript

### PR DESCRIPTION
**Pull request by alfoa**
--------
Pull Request Description
--------
##### What issue does this change request address? (Use "#" before the issue to link it, i.e., #42.)
Ref. #655 

##### What are the significant changes in functionality due to this change request?
This PR add a script to reformat the code and the docstrings to a RAVEN standard.
It can be used as utility function by the developers to be sure that modules and files are 
consistent with the Standard

----------------
For Change Control Board: Change Request Review
----------------
The following review must be completed by an authorized member of the Change Control Board.
- [x] 1. Review all computer code.
- [x] 2. If any changes occur to the input syntax, there must be an accompanying change to the user manual and xsd schema. If the input syntax change deprecates existing input files, a conversion script needs to be added (see Conversion Scripts).
- [x] 3. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://github.com/idaholab/raven/wiki/RAVEN-Code-Standards#python) for details.
- [x] 4. Automated Tests should pass, including run_tests, pylint, manual building and xsd tests. If there are changes to Simulation.py or JobHandler.py the qsub tests must pass.
- [x] 5. If significant functionality is added, there must  be tests added to check this. Tests should cover all possible options.  Multiple short tests are preferred over one large test. If new development on the internal JobHandler parallel system is performed, a cluster test must be added setting, in <RunInfo> XML block, the node ```<internalParallel>``` to True.
- [x] 6. If the change modifies or adds a requirement or a requirement based test case, the Change Control Board's Chair or designee also needs to approve the change.  The requirements and the requirements test shall be in sync.
- [x] 7. The merge request must reference an issue.  If the issue is closed, the issue close checklist shall be done.
- [x] 8. If an analytic test is changed/added is the the analytic documentation updated/added?

### CODE REVIEW

**Review by PaulTalbot-INL — DISMISSED** [review]
A few comments with quite minor improvement requests.

File: scripts/fix_raven_docstrings.py [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+
+#External Modules--------------------
+import sys
+import os
+#External Modules--------------------
+
+def trimQuoteDocstring(docstring, quoteIndent, maxColumns=120):
+  """
+    This method is aimed to trim to the quote docstrings to the
+    be limited in the maxColumns number
```
[the remark this answers is not in the document]
Comment (aalfonsi) [reply scripts/fix_raven_docstrings.py:35]: fixed

**Comment by aalfonsi** [comment]
@PaulTalbot-INL @wangcj05 @mandd @joshua-cogliati-inl

File: scripts/fix_raven_docstrings.py [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+        if len(lineToAdd+word)+3  < maxColumns:
+          lineToAdd = lineToAdd +" "+word
+        else:
+          newLines.append(lineToAdd+"\n")
+          lineToAdd = " "*breakIndex+" "+word
+        if i == len(words)-1:
+          newLines.append(lineToAdd+"\n")
+      outputLines+=newLines
+    else:
+      outputLines.append(line)
```
Comment (wangcj05) [inline scripts/fix_raven_docstrings.py:73]: why use 'line' here? not 'indentedLine'?

File: scripts/fix_raven_docstrings.py [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+      elif "@ out" in line.lower() or '@out' in line.lower():
+        outIndex.append(cnt)
+  for cnt, line in enumerate(lines):
+    indentedLine = " "*(quoteIndent+2) + line.lstrip()
+    if cnt in inIndex+outIndex:
+      indentedLine = indentedLine.replace("@O","@ O").replace("@I","@ I").replace("@o","@ O").replace("@i","@ I")
+      # the following is scannint the string to find the indeces of the
+      # commas (in the RAVEN standard)
+      breakIndex = [i for i, ltr in enumerate(indentedLine) if ltr == ',']
+      breakIndex = quoteIndent+8
```
Comment (wangcj05) [inline scripts/fix_raven_docstrings.py:53]: What is the purpose for line 52? 'breakIndex' is reassigned in line 53.

**Review by wangcj05 — CHANGES_REQUESTED** [review]
some minor comments for your consideration

**Comment by aalfonsi** [comment]
@wangcj05  addressed

**Comment by aalfonsi** [comment]
@wangcj05 @joshua-cogliati-inl this is ready to be merged

**Comment by aalfonsi** [comment]
@joshua-cogliati-inl @wangcj05 I also fixed the doc issue that presumably made "development" doc step fail

**Review by wangcj05 — APPROVED** [review]
This PR looks good to me.

**Comment by wangcj05** [comment]
PR is good, and checklist is satisfied. PR can be merged.

**Comment by wangcj05** [comment]
@aalfonsi Thanks for your contribution, I have merged your PR.

**Pull request merged**